Runtime/Scripts/RTCPeerConnection.cs

```diff
@@ -124,19 +154,46 @@ public DelegateOnTrack OnTrack
             set
             {
                 onTrack = value;
-                selfOnTrack = new DelegateNativeOnTrack(PCOnTrack);
+                selfOnTrack = PCOnTrack;
                 NativeMethods.PeerConnectionRegisterOnTrack(self, selfOnTrack);
             }
         }
 
+        internal DelegateSetSessionDescSuccess OnSetSessionDescriptionSuccess
+        {
+            get => onSetSessionDescSuccess;
+            set
+            {
+                onSetSessionDescSuccess = value;
+                selfOnPeerConnectionSetSessionDescSuccess = OnSetSessionDescSuccess;
```

Comment:
Do we need to assign to `selfOnPeerConnectionSetSessionDescSuccess` and `selfOnPeerConnectionSetSessionDescFailure `?

Reply:
I removed it